src/Sulu/Bundle/AdminBundle/Resources/config/services.xml

```diff
@@ -121,5 +121,19 @@
         <service id="sulu_admin.schema_handler" class="Sulu\Bundle\AdminBundle\Serializer\Handler\SchemaHandler">
             <tag name="jms_serializer.subscribing_handler"/>
         </service>
+
+        <service id="sulu_admin.dropdown_toolbar_action_handler"
```

Comment:
would be more specific with the id e.g. `sulu_admin.serializer_subscriber.dropdown...`

Reply:
Renamed the service to be callled `subscriber` at the end. Was experimenting with handlers, but that didn't work out well...

I am against introducing sub "namespaces", as this was already a mess in the old codebase. Just having the bundle prefixed with a dot seems like the cleaner solution to me.